easybuild/easyconfigs/m/MATLAB/MATLAB-2018a.eb

```diff
@@ -0,0 +1,17 @@
+name = 'MATLAB'
+version = '2018a'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# be sure to copy both DVD content to the SAME directory,
+# including the hidden files, especially .dvd1 and .dvd2
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8.0_172')]
```

Comment:
@cstackpole you can avoid the test failure by using the Java wrapper:
`dependencies = [('Java', '1.8')]`